src/utils.cpp

```diff
@@ -179,6 +179,28 @@ namespace sdk {
 #endif
     }
 
+    std::string extract_tx_from_psbt(const std::string& psbt)
+    {
+#ifdef BUILD_GDK_RUST
+        return ga_rust::extract_tx_from_psbt(b2h(base64_to_bytes(psbt)));
```

Comment:
we should avoid these extra conversions to bytes then hex and pass the base64 to/from rust

Reply:
Ideally yes, however if we need to do the b64 conversions in rust, we need to add a new dependency there.
Since we expect that PSET/PSBT operations will soon be done by wally on the cpp side, to me it does not seem worth adding a new dependency.